library/account/src/main/kotlin/org/cru/godtools/account/GodToolsAccountManager.kt

```diff
@@ -33,13 +37,17 @@ class GodToolsAccountManager @VisibleForTesting internal constructor(
     @get:VisibleForTesting
     internal val providers: List<AccountProvider>,
     private val userApi: Lazy<UserApi>,
+    @ApplicationContext
```

Comment:
since this isn't the constructor yo are injecting, you don't need to specify the `@ApplicationContext` annotation on it